### PR TITLE
bring to master

### PR DESCRIPTION
**Pull request by nileshcs**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ simplifies the operation and significantly lowers the cost of Loki.
 2. [Installation](installation/README.md)
     1. [Installing with Tanka](installation/tanka.md)
```
Comment (oddlittlebird) [inline docs/README.md:20]: Please change these links to lazy numbering

File: docs/clients/fluentd/README.md [hunk 2]
```diff
@@ -119,18 +119,20 @@ This plugin automatically adds a `fluentd_thread` label with the name of the buf
 
 ## Docker Image
```
Comment (oddlittlebird) [inline docs/clients/fluentd/README.md:120]:
```suggestion
## Docker image
```

File: docs/clients/fluentd/README.md [hunk 3]
```diff
@@ -119,18 +119,20 @@ This plugin automatically adds a `fluentd_thread` label with the name of the buf
...
 
-There is a Docker image `grafana/fluent-plugin-grafana-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/master/fluentd/fluent-plugin-grafana-loki/docker/conf). By default, fluentd containers use the configurations but you can also specify your `fluentd.conf` with `FLUENTD_CONF` environment variable.
+There is a Docker image `grafana/fluent-plugin-loki:master` which contains [default configuration files](https://github.com/grafana/loki/tree/master/fluentd/fluent-plugin-loki/docker/conf). By default, fluentd containers use the configurations but you can also specify your `fluentd.conf` with `FLUENTD_CONF` environment variable.
```
Comment (oddlittlebird) [inline docs/clients/fluentd/README.md:122]: Is Fluentd a proper name? It is lowercase here and capitalized below. Please capitalize consistently.

File: docs/clients/fluentd/README.md [hunk 4]
```diff
@@ -119,18 +119,20 @@ This plugin automatically adds a `fluentd_thread` label with the name of the buf
...
 
 This image also uses `LOKI_URL`, `LOKI_USERNAME`, and `LOKI_PASSWORD` environment variables to specify the Loki's endpoint, user, and password (you can leave the USERNAME and PASSWORD blank if they're not used).
 
+This image will start an instance of Fluentd to forward incoming logs to the specified Loki url. As an alternate, containerized applications can also use [docker driver plugin](../docker-driver/README.md) to ship logs without needing Fluentd.
```
Comment (oddlittlebird) [inline docs/clients/fluentd/README.md:126]:
```suggestion
This image will start an instance of Fluentd to forward incoming logs to the specified Loki URL. Applications run in containers can also use [Docker driver plugin](../docker-driver/README.md) to ship logs without needing Fluentd.
```

**Comment by nileshcs** [comment]
Sorry the PR was created as a mistake; closed it and will create the proper one next.

**Pull request closed without merging**